src/test/java/com/wikia/webdriver/testcases/discussions/GuidelinesTests.java

```diff
@@ -64,6 +64,16 @@ public void staffOnDesktopCanClickEditGuidelines() {
         Assertion.assertTrue(guidelinesPage.isModalGuidelinesDisplayed());
     }
 
+    @Test(groups = "discussions-discussionsModeratorOnDesktopCanClickEditGuidelines")
+    @Execute(asUser = User.DISCUSSIONS_MODERATOR)
+    @InBrowser(browser = Browser.FIREFOX, browserSize = DESKTOP_RESOLUTION)
+    public void discussionsModeratorOnDesktopCanClickEditGuidelines() {
+        GuidelinesPage guidelinesPage = new GuidelinesPage();
+        guidelinesPage.open();
+
+        Assertion.assertTrue(guidelinesPage.isEditButtonDisplayed());
```

Comment:
Test is checking only that Edit button is displayed/visible. But in name of class there is info Moderator On Desktop Can Click Edit Guidelines. Rename test or change way of checking functionality.